scripts/host-check

```diff
@@ -1486,10 +1493,10 @@ def perform_checks(checks: List[Check]) -> bool:
                     status = CheckState.SUCCESS
                     msg = None
             except CmdTimeoutError as e:
-                status = CheckState.WARNING
-                msg = f"Unexpected error: {e}"
+                status = CheckState.ERROR
+                msg = f"Command timed out: {e}"
             except Exception as e:
-                status = CheckState.FAILED
+                status = CheckState.ERROR
                 msg = f"Unexpected error: {e}"
```

Comment:
Sorry, after looking at the testcase output (e.g. `TestCPUCores.test_timeout_error()`) where it says "Command timed out: Timed out while executing command:", I think it would read better to keep the "Unexpected error" message. So there's actually no need to handle `CmdTimeoutError` separately, this can just be
```python
except Exception as e:
    status = CheckState.ERROR
    msg = f"Unexpected error: {e}"